apps/ui-builder/src/components/InsertComponent.tsx

```diff
@@ -13,25 +34,22 @@ export const InsertComponent = ({ onComponentAdded, onDismiss }) => {
     onDismiss && onDismiss();
   }, [onDismiss]);
 
+  const onChange = React.useCallback((e, data) => setSelectedComponent(String(data.value)), [setSelectedComponent]);
+
   const items = Object.keys(componentInfoContext.byDisplayName);
 
   return (
-    <Dialog
-      header="Insert component"
-      confirmButton="Insert"
-      cancelButton="Cancel"
-      open={true}
-      content={
-        <Dropdown
-          search
-          highlightFirstItemOnOpen
-          placeholder="Choose component"
-          items={items}
-          onChange={(e, data) => setSelectedComponent(String(data.value))}
-        />
-      }
-      onConfirm={confirm}
-      onCancel={dismiss}
-    />
+    <Dialog open={true}>
+      <DialogSurface className={styles.surface}>
+        <h2 className={styles.header}>Insert component</h2>
```

Comment:
```suggestion
        <DialogTitle>Insert component</DialogTitle>
```
This way, dialog will be properly titled (using aria-labelledby pointing to the title) and style will not be necessary